lang/snobol4/Portfile

```diff
@@ -3,30 +3,37 @@
 PortSystem          1.0
 
 name                snobol4
-version             1.2
+version             2.3.1
 revision            0
 categories          lang
 license             BSD
-maintainers         nomaintainer
+maintainers         regressive.org:snobol4
 platforms           darwin
-homepage            http://www.snobol4.org/
-master_sites        ftp://ftp.ultimate.com/snobol
+homepage            http://www.regressive.org.org/snobol4
+master_sites        ftp://ftp.regressive.org/snobol \
+                    http://ftp.regressive.org/snobol
 
 description         SNOBOL4 programming language
 
 long_description    SNOBOL4 (StriNg Oriented and symBOlic Language) is \
                     a language for text processing, pattern matching, \
                     and much more, first designed and implemented at \
-                    Bell Telephone Laboratories, Inc. (BTL) in the 1970's.
-
-checksums           rmd160  b53b6cfe985f402143b91e0c7cdd1b7316700b7e \
-                    sha256  bc3b1a644a966dc8121590495e2835252c7ff4e22f138c1cd96a19ed57f4f5eb \
-                    size    658128
-
-patchfiles          configure.patch
-
-configure.args      --incdir=${configure.sdkroot}/usr/include \
-                    --libdir=${configure.sdkroot}/usr/lib \
-                    --mandir=${prefix}/share/man
+                    Bell Telephone Laboratories, Inc. (BTL) in the 1970's. \
+                    Includes SPITBOL, Blocks and other extensions.
+
+checksums           rmd160  a05be6f3d02005deaae6d9b00763aa99f4160a2e \
+                    sha256  91244d67d4e29d2aadce5655bd4382ffab44c624a7ea4ad6411427f3abf17535 \
+                    size    986906
+
+# Arbitrary list of dependencies: declare variants?
+# Will use openssl if available, but can't be told where to look
+# for a specific version.
+depends_lib         port:libffi \
+                    port:sqlite3 \
+                    port:readline \
+                    port:zlib
```

Comment:
My configuration system is opportunistic, and --without is on my TODO list.

Right now it prefers ndbm.h and routines in libc. If Apple should ever removes classic ndbm from their libc, it will attempt to find a different provider for the ndbm API (with, likely, another file format).  So should I pick a different provider (and possibly break existing programs)?

I was probably looking at SSL support when I wrote the comment.  I saw a frightening amount of goo in other Portfiles to deal with different OpenSSL versions.  Is there a document that describes best practices?  My experience is that OpenSSL has deprecated API's in recent history, and to defend against that one might be tempted to request an exact version of SSL, but that seems painful/dangerous/unworkable!

Reply:
ideally, the only thing you should need to do is add the line below near the top of the Portfile: ```PortGroup           openssl 1.0```. 

Until the time you have the `--without``` configure arguments implemented, please just add all the (optional) depdencencies.